<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="4" hostname="Umas-MBP.attlocal.net" failures="1" tests="3" name="Sanity" time="0.002" errors="0" timestamp="2024-02-17T21:11:45 PST">
  <testcase classname="testSuite1.Module2" name="test001" time="0.0"/>
  <testcase classname="testSuite1.Module2" name="test003" time="0.0"/>
  <testcase classname="testSuite1.Module1" name="test001" time="0.0">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testSuite1.Module1.test001(Module1.java:22)
at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- test001 -->
  <testcase classname="testSuite1.Module1" name="test002" time="0.0">
    <ignored/>
  </testcase> <!-- test002 -->
  <testcase classname="testSuite1.Module1" name="test003" time="0.0">
    <ignored/>
  </testcase> <!-- test003 -->
  <testcase classname="testSuite1.Module1" name="login" time="0.0">
    <ignored/>
  </testcase> <!-- login -->
  <testcase classname="testSuite1.Module2" name="test002" time="0.0">
    <ignored/>
  </testcase> <!-- test002 -->
</testsuite> <!-- Sanity -->
